### ISSUE TITLE
Fix `declaration-property-value-no-unknown` parseError even when a file starts with `/* stylelint-disable */`

### ISSUE THREAD

**Issue by romainmenke** [stylelint/stylelint#6649]
### What steps are needed to reproduce the bug?

```css
/* stylelint-disable */
a {
  filter: alpha(opacity=30);
}
```

CSS Tree is unable to parse old CSS hacks.
It also isn't designed to handle all these things.

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "declaration-property-value-no-unknown": true
}
```

### How did you run Stylelint?

CLI and VSCode

### Which version of Stylelint are you using?

15.1.0

### What did you expect to happen?

I expected to be able to ignore very old CSS with `/* stylelint-disable */` at the top a file.

### What actually happened?

An error was emitted.

### Proposal to fix the bug

https://github.com/stylelint/stylelint/blob/main/lib/rules/declaration-property-value-no-unknown/index.js#L93-L99

```js
try {
	cssTreeValueNode = parse(value, { context: 'value' });

	if (containsUnsupportedFunction(cssTreeValueNode)) return;
} catch (e) {
	result.warn(`Cannot parse property value "${value}"`, {
		node: decl,
		stylelintType: 'parseError',
	});

	return;
}
```

When I use `report()` I am able to use `disable` comments to suppress parse errors.

```js
try {
	cssTreeValueNode = parse(value, { context: 'value' });

	if (containsUnsupportedFunction(cssTreeValueNode)) return;
} catch (e) {
	const index = declarationValueIndex(decl);
	const endIndex = index + decl.value.length;

	report({
		message: messages.parseError(prop, decl.value),
		node: decl,
		index,
		endIndex,
		result,
		ruleName,
	});

	return;
}
```

**Comment by jeddy3** [comment]
@romainmenke Thanks for the report.

Your proposal LGTM.

I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.
